settings-boundary/RootSettings.tsx

```diff
@@ -0,0 +1,179 @@
+import { IfAuthenticated, UserSession } from "@user/Session"
+import { StyleProp, ViewStyle, StyleSheet, View } from "react-native"
+import { SettingsScrollView } from "./components/ScrollView"
+import { SettingsCardSectionView } from "./components/Section"
+import { useState } from "react"
+import { useConst } from "@lib/utils/UseConst"
+import { SettingsNavigationLinkView } from "./components/NavigationLink"
+import { AppStyles } from "@lib/AppColorStyle"
+import { BodyText } from "@components/Text"
+import { nativeBuildVersion } from "expo-application"
+import { SettingsCardView } from "./components/Card"
+import Animated, { FadeIn } from "react-native-reanimated"
+
+export type RootSettingsScreenLinkID =
+  | "calendarSettings"
+  | "generalSettings"
+  | "privacySettings"
+  | "eventSettings"
+  | "helpAndSupport"
+  | "notificationsSettings"
+  | "appearanceSettings"
+  | "blockList"
+
+export type RootSettingsProps = {
+  areNotificationsEnabled: boolean
+  onSettingsScreenLinkTapped: (id: RootSettingsScreenLinkID) => void
+  onAccountInfoLinkTapped: (session: UserSession) => void
+  style?: StyleProp<ViewStyle>
+}
+
+export const RootSettingsView = ({ style, ...props }: RootSettingsProps) => (
+  <SettingsScrollView style={style}>
+    <MainScreenLinksSection {...props} />
+    <SupportSectionView
+      onSettingsScreenLinkTapped={props.onSettingsScreenLinkTapped}
+    />
+  </SettingsScrollView>
+)
+
+type MainScreenLinksSectionProps = Omit<RootSettingsProps, "style">
+
+const MainScreenLinksSection = ({
+  onSettingsScreenLinkTapped,
+  onAccountInfoLinkTapped,
+  areNotificationsEnabled
+}: MainScreenLinksSectionProps) => (
+  <SettingsCardSectionView>
+    <SettingsNavigationLinkView
+      iconName="settings"
+      iconBackgroundColor={AppStyles.black}
+      title="General"
+      onTapped={() => onSettingsScreenLinkTapped("generalSettings")}
+    />
+    <IfAuthenticated
+      thenRender={(session: UserSession) => (
+        <SettingsNavigationLinkView
+          iconName="person"
+          iconBackgroundColor={AppStyles.blue}
+          title="Account Info"
+          onTapped={() => onAccountInfoLinkTapped(session)}
+        />
+      )}
+    />
+    <SettingsNavigationLinkView
+      iconName="color-palette"
+      iconBackgroundColor={AppStyles.red}
+      title="Appearance"
+      onTapped={() => onSettingsScreenLinkTapped("appearanceSettings")}
+    />
+    <SettingsNavigationLinkView
+      iconName="golf"
+      iconBackgroundColor={AppStyles.green}
+      title="Events"
+      onTapped={() => onSettingsScreenLinkTapped("eventSettings")}
+    />
+    <SettingsNavigationLinkView
+      iconName="notifications"
+      iconBackgroundColor={AppStyles.yellow}
+      title="Notifications"
+      rightAccessory={
+        <BodyText style={styles.notificationsText}>
+          {areNotificationsEnabled ? "On" : "Off"}
```

Comment:
Can we call it "areSystemNotificationsEnabled", I was unclear at first which notification this was referring to